libbeat/publisher/pipeline/pipeline.go

```diff
@@ -71,10 +71,6 @@ type Pipeline struct {
 	waitCloseTimeout time.Duration
 	eventWaitGroup   *sync.WaitGroup
 
-	// closeRef signal propagation support
-	guardStartSigPropagation sync.Once
```

Comment:
These variables aren't related to producer cancel, but they caused a linter error because they're unused.